lib/biran/exceptions.rb

```diff
@@ -0,0 +1,16 @@
+module Biran
+  class ConfigSyntaxError < ::StandardError
+    def initialize(msg='Missing required argument or bad formatting in config file')
```

Comment:
do you want to be able to have  a `nil` msg? if the answer is no, you should use below

`@msg ||= 'Missing required argument or bad formatting in config file'`  instead and set the default to `nil`

Reply:
I don't know if I do, so because I don't know, I will add the option. Doesn't hurt anything.

Reply:
@javierg  is the proper way to do this like the following:
```
@msg = msg ||= 'Missing required argument or bad formatting in config file'
```
If I didn't inlcude `= msg ||= ` in there, then any message I did send got ignored because it wasn't referenced.

Reply:
right actually you will be more like

`@msg = msg || 'Missing required argument or bad formatting in config file'`

you don't need a local var for `msg`